chore: sort class list

yoshi-automation opened the pull request:
This PR was generated using Autosynth. :rainbow:

Synth log will be available here:
https://source.cloud.google.com/results/invocations/21202808-e3e7-484e-a1fa-d5b954358bde/targets

- [ ] To automatically regenerate this PR, check this box.

tswast commented on .kokoro/trampoline_v2.sh:
```diff
@@ -159,7 +159,7 @@ if [[ -n "${KOKORO_BUILD_ID:-}" ]]; then
 	"KOKORO_GITHUB_PULL_REQUEST_NUMBER"
 	"KOKORO_GITHUB_PULL_REQUEST_COMMIT"
-	# For FlakyBot
+	# For Build Cop Bot
```
out-of-date. Maybe needs https://github.com/googleapis/python-datacatalog/pull/120 first

yoshi-automation committed:
changes without context

        autosynth cannot find the source of changes triggered by earlier changes in this
        repository, or by version upgrades to tools such as linters.
